service_discovery_manager/Project/Android/app/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ add_library( # Sets the name of the library.
         ../../../Component/mmBASE/BaseAPI/bSocket.cpp
         ../../../Component/mmBASE/BaseAPI/bTask.cpp
         ../../../Component/mmBASE/BaseAPI/bThread.cpp
+        ../../../Component/mmBASE/BaseAPI/ifaddrs.cc
```

Comment:
Please add source file to Component/SOURCE.LIST as well.

Reply:
done